cmd/cloud-controller-manager/app/apis/config/v1alpha1/defaults.go

```diff
@@ -47,5 +47,5 @@ func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManager
 	// Use the default RecommendedDefaultGenericControllerManagerConfiguration options
 	kubectrlmgrconfigv1alpha1.RecommendedDefaultGenericControllerManagerConfiguration(&obj.Generic)
 	// Use the default RecommendedDefaultServiceControllerConfiguration options
-	serviceconfigv1alpha1.RecommendedDefaultServiceControllerConfiguration(&obj.ServiceController)
+	cmconfigv1alpha1.RecommendedDefaultServiceControllerConfiguration(&obj.ServiceController)
```

Comment:
I expected the defaulting functions for the service controller to accompany the service controller config type and be in `k8s.io/cloud-provider`

Reply:
+1

Reply:
Updated.